### PR TITLE
[Doc Issue][Updated the doc to remove the GUID][3025186]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Review by Prasad-MSFT — CHANGES_REQUESTED** [review]
Would be better to remove the random id from code snippet as well.
image

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/35b1867a7235d69d2d0aeba0e724d629e8127ba8/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/573fb28b-40ab-497d-806f-817392f82666) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ab855f63d3b113e4dfd5398b2b9c98660766573f/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3d88ead1-25a5-4a68-8866-319cbb70f3b5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Prasad-MSFT — CHANGES_REQUESTED** [review]
Please remove an extra space 
image
as well as remove the random id from TeamsJSv1 as well
image

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c7c21495e87ebcb22e445254c2c989cf1a5d7f13/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4c77d8ec-c338-4b88-90cd-26045cb6299f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by v-bvishnu** [comment]
Hi Prasad,

Extra space and GUID from TeamsJS V1 is removed, please check and let me know.

**Review by Prasad-MSFT — CHANGES_REQUESTED** [review]
You removed entire app id and didn't replace with place holder. Please add as below:
```
// Open an app install dialog from your tab
microsoftTeams.executeDeepLink("https://teams.microsoft.com/l/app/{appId}");
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2e32f6e2ade2ed301a8703d11c6ada230baad116/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cedf0201-0e7e-4e09-b252-1835607f12bf) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c5bd7e3315328e3315cd5f2d4c3a4f4812478e62/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4e9a782e-0952-473d-9ea2-b084a0a07e47) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Prasad-MSFT — DISMISSED** [review]
Changes look fine

File: msteams-platform/concepts/build-and-test/deep-link-application.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Applications can use the TeamsJS library to open the dialog without manually gen
 ```javascript
 // Open an app install dialog from your tab
 if(appInstallDialog.isSupported()) {
-    const dialogPromise = appInstallDialog.openAppInstallDialog({ appId: "f46ad259-0fe5-4f12-872d-c737b174bcb4" });
+    const dialogPromise = appInstallDialog.openAppInstallDialog(<appId>);
```
Comment (Wajeed-msft) [inline msteams-platform/concepts/build-and-test/deep-link-application.md:33]:
```suggestion
    const dialogPromise = appInstallDialog.openAppInstallDialog({ appId: "<appId>" });
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/02fa68d7896634480469c94567784aface35e817/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0e6dcd4-61df-4ed7-8dad-efe0ee50e8fc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e3db755ad15433d57c58ec4f6e2c03c6666a5c40/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/54cb0f97-41f3-4e97-b54e-06e04f997750) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/96a63334b2d75dd6c2e13cf1bb0312e40efdb6f6/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6a42fdc8-5160-460e-a5b2-3bc6ac56c8ed) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a20036a291369547231ea7e956ae92b3c796954/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c2e489f4-55a1-4b7b-ab29-fc79248086bc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-link-application.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/concepts/build-and-test/deep-link-application.md]: Updated as per comments provided.

**Review by Wajeed-msft — APPROVED** [review]
@AE-MS - could you please take a look at the changes?

**Review by v-habibaumme — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/97d95959e489faba71dbccf9e61aebd218b2c5de/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 94 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4754fa07-0d5c-4714-ad6a-8fb25d49dcfa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-application.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d52fec8069b4b9535b48d224e32f29877cb189a2/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-application.md) | 94 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3177a73b-7108-45da-bb44-191a2da4ffe1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by AE-MS** [comment]
> AE-MS

Guess it doesn't matter what I say since it's already merged but it looks okay to me.  😁

**Pull request merged**